### PR TITLE
feat: Added Dutch translation

### LINKED ISSUE #398: Translate app content into Dutch
Add Dutch translations to the app

Related to #21 #224

### PR DESCRIPTION
**Pull request by jpelgrim**
## What does this PR accomplish?

Added Dutch translation

Closes #398 

## Did you add any dependencies?

No

## How did you test the change?

Compiled and run the app. Intl-strings are not used yet in the app I see, so I couldn't add screenshots. I just added Dutch translations to the strings that were already defined.

### CODE REVIEW

File: client/flutter/lib/generated/intl/messages_en.dart [hunk 1]
```diff
@@ -27,7 +27,7 @@ class MessageLookup extends MessageLookupByLibrary {
...
     "childLove" : MessageLookupByLibrary.simpleMessage("Children need adults\' love and attention during difficult times. Give them extra time and attention.\nRemember to listen to your children, speak kindly and reasure.\nIf possible, make opportunities for the child to play and relax."),
     "childRoutines" : MessageLookupByLibrary.simpleMessage("Keep to regular routines and schedules as much as possible, or help create new ones in a new environment, including school/learning as well as time for safely playing and relaxing."),
-    "childStress" : MessageLookupByLibrary.simpleMessage("Children may respond to stress in different ways such as being more clingy, anxious, withdraw, angry or agitated, bedwetting etc\nRespind to your child\'s reactions in a supportive way, listen to their concerns and give them extra love and attention."),
+    "childStress" : MessageLookupByLibrary.simpleMessage("Children may respond to stress in different ways such as being more clingy, anxious, withdraw, angry or agitated, bedwetting etc\nRespond to your child\'s reactions in a supportive way, listen to their concerns and give them extra love and attention."),
```
Comment (SamMousa) [inline client/flutter/lib/generated/intl/messages_en.dart]: Please do this in a separate PR, PR with just a fixed typo can easily be merged.

Comment (jpelgrim) [reply client/flutter/lib/generated/intl/messages_en.dart]: Removed in this PR

Comment (jpelgrim) [reply client/flutter/lib/generated/intl/messages_en.dart]: Created PR #404 for the typo

**Comment by sapte91** [comment]
@jpelgrim Thank you so much for your support of this project and for the time you spent on this PR - we’re so humbled by your willingness to jump in! We have an important update on localization for the application: We’ll primarily be sourcing and managing translations from Lingo24, our translation management system. Also, our designs changed, so these translations are now out of date. As such, I’m closing this PR.

We will still need translation volunteers - as our set of supported languages and locales grows, there will be plenty of opportunity to contribute! Our projected needs for the next two months are in native speaker LQA and translations for languages that we cannot support on Lingo24. We apologize that we are still in the early process of figuring out how to manage all the contributions and will have more details soon. Until then, please fill out the volunteer form below and we’ll get back in touch shortly.

https://forms.gle/DTqvaqh4mE7bDe5U7
